docs/common-coreboot-docs/dasharo_tools_suite.md

```diff
@@ -162,6 +181,150 @@ fwupdmgr refresh
 fwupdmgr update
 ```
 
+## Dasharo EC Transition
+
+DTS allows to perform full EC transition from vendor to Dasharo/EC FW. For now
+this functionality is supported on NS50 and NS70 laptops.
+
+To perform EC transition make sure you are running DTS version 1.0.1 or higher
+and follow this steps:
+
+* After boot, choose option number 9 to drop to Shell.
+* Plug in power supply, without it, flashing EC is not possible as losing power
+  may cause in damaged firmware.
+* Download BIOS and EC update files.
+
+```bash
+wget -O firmware.rom https://cloud.3mdeb.com/index.php/s/SKpqSNzfFNY7AbK/download
```

Comment:
maybe in this command, we should add an output parameter? During testing, I noticed that downloaded files are saved as `download` without any information about the format of the file.

Reply:
This is way we use here `-O` flag - this command saves file to `firmware.rom`